flare/md.py

```diff
@@ -34,3 +35,61 @@ def calculate_temperature(new_pos, structure, dt, noa):
     temperature = 2 * KE / ((3 * noa - 3) * kb)
 
     return KE, temperature, velocities
+
+
+def get_random_velocities(noa: int, temperature: float, mass: float):
+    """Draw velocities from the Maxwell-Boltzmann distribution, assuming a
+    fixed mass for all particles in amu.
+
+    Args:
+        noa (int): Number of atoms in the system.
+        temperature (float): Temperature of the system.
+        mass (float): Mass of each particle in amu.
+
+    Returns:
+        np.ndarray: Particle velocities, corrected to give zero center of mass motion.
+    """
+
+    mass_md = mass * 0.000103642695727
```

Comment:
What is this number? Can you please make it a named variable?

Reply:
Added comments explaining them.